tensorboard/plugins/npmi/summary.py

```diff
@@ -111,3 +111,34 @@ def npmi_values(tensor, step=None, description=None):
             step=step,
             metadata=metadata.create_summary_metadata(description),
         )
+
+
+def npmi_embeddings(tensor, step=None, description=None):
+    """Write the actual npmi values.
```

Comment:
Could you clarify how this differs from `npmi_values()` where the one-line docstring is also "Write the actual npmi values"?  The shape of the tensor is different, but a bit of further detail about the semantic difference here would be useful.

Reply:
Sure, forgot to update this comment. Did that now.